README.md

```diff
@@ -768,6 +768,7 @@ Libraries to help manage database schemas and migrations.
 * [PHP Queue](https://github.com/CoderKungfu/php-queue) - An application for managing queueing backends.
 * [MailCatcher](https://github.com/sj26/mailcatcher) - A web tool for capturing and viewing emails.
 * [Cachet](https://github.com/cachethq/cachet) - The open source status page system.
+* [phpRedisAdmin](https://github.com/ErikDubbelboer/phpRedisAdmin) - A simple web interface to manage [Redis](http://redis.io/) databases.
```

Comment:
I was under the impression that all stand-alone web apps should go under 'Software'/'Web applications'. If it would be a library then I would agree.

Maybe I did not understood correctly? If that's the case, then sure, we can move it to miscellaneous.